src/live.rs

```diff
@@ -43,8 +44,11 @@ pub fn iso_remove(config: &IsoIgnitionRemoveConfig) -> Result<()> {
 }
 
 pub fn iso_ignition_embed(config: &IsoIgnitionEmbedConfig) -> Result<()> {
-    let mut holder = CopiedFileHolder::new(&config.input, config.output.as_ref())?;
-    let mut embed = EmbedArea::for_file(&mut holder.file)?;
+    if config.output.as_deref() == USE_STDOUT
```

Comment:
Maybe do this test once, as `let use_stdout = config.output.as_deref() == Some("-");`?